service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersAboutToSubmitControllerTest.java

```diff
@@ -216,6 +219,7 @@ void shouldBuildNewBlankOrderForClosedCase() {
                     .build())
                 .dateTimeIssued(now())
                 .approvalDate(dateNow())
+                .markedFinal(NO.getValue())
```

Comment:
should one of these tests cover `YES.getValue()` ?

Reply:
the markedFinal is a passthrough value at this point of the flow in the about to submit. The real value gets set during the is final order question